gnosis/conditional-markets-interface
# Feature #58: Graph scalar resolution value

# Description
This implements a new "Reference Line" to display the winning outcome for scalar markets on the graph.

# To Test
- Visit an unresolved scalar market, the winning outcome should not be displayed
- Visit a resolved scalar market, observe the green reference line with the correctly aligned tooltip, showing the winning outcome.

# Code
There's no way we can add any more lines to this graph. It's becoming chaos. 🙈
